# AoCS0002

## Title
Invalid `PartSolutionStatus` value on `PartSolutionAttribute`

## Category
Validity

## Severity
Error

## Details
This error is reported when the `PartSolutionStatus` value in a `PartSolutionAttribute` is not a valid value.

## Example
```csharp
public class Day10 : Problem<int>
{
    [PartSolution((PartSolutionStatus)456)] // AoCS0002 will appear here
    public int SolvePart1() => NotSolution();
}
```
