# AoCS0005

## Title
Denoted problem year must be a valid year after 2015, up until the current one

## Category
Convention

## Severity
Error

## Details
This error is reported when a problem solution namespace container denotes an invalid year. Valid years range from 2015 up until the current year at the time of compilation (does not require the month to be December).

## Example
```csharp
namespace AoC.Problems.Year2099; // AoCS0005 will appear here
```
