# AoCS0001

## Title
Invalid `PartSolutionAttribute` on non-solution function

## Category
Validity

## Severity
Error

## Details
This error is reported when the `PartSolutionAttribute` is applied on a function that does not define the solution for a problem part.

## Example
```csharp
public class Day10 : Problem<int>
{
    [PartSolution(PartSolutionStatus.WIP)]
    public int SolvePart1() => NotSolution();

    [PartSolution(PartSolutionStatus.WIP)] // AoCS0001 will appear here
    public void NotSolution() => 1;
}
```
